factories/__init__.py

```diff
@@ -0,0 +1,2 @@
+import factory
+factory.Faker._DEFAULT_LOCALE = 'es_ES'
```

Comment:
If we want to make it public, the locale maybe should be an environment var ?

Reply:
Better not to tamper the default locale at all! Just let any usage of this library be to choose how they work it out.